app/javascript/widget/components/PreChat/Form.vue

```diff
@@ -114,7 +114,7 @@ export default {
       return this.options.preChatMessage;
     },
     preChatFields() {
-      return this.disableContactFields ? [] : this.options.preChatFields;
+      return this.options?.preChatFields || [];
```

Comment:
In this case, I see that the option `disableContactFields` comes from the parent component "PreChatForm", which infers the data from `$route.params` 

Is there an case where the value is used except when the user is returning? Just want to ensure there are no unwanted side effects of ignoring that. 

If there aren't we could remove that check anyway, since it's not used